index.html

```diff
@@ -63,7 +63,7 @@ <h2 class="who-am-i">I’m a software developer</h2>
             
          
 
-        </section> 
+        </div> 
```

Comment:
change back to section tag to align with the use of good sematics

Reply:
- [ ] Please be as polite to the student when asking for changes and not sound commanding. In this case, you're commanding the student and that's not cool.
- [ ] Use polite words like: "Please do this ....", "Kindly use this ...", "It would be nice if you use this ....",